hummingbot/strategy/pure_market_making/pure_market_making_v2.pyx

```diff
@@ -346,6 +363,31 @@ cdef class PureMarketMakingStrategyV2(StrategyBase):
             object market_info = self._sb_order_tracker.c_get_market_pair_from_order_id(order_id)
             LimitOrder limit_order_record
 
+        # Replenish Delay and order filled stop cancellation are only for single order mode
+        # (identified using name of the delegate)
+        if self.sizing_delegate_name in self.SINGLE_ORDER_SIZING_DELEGATES:
```

Comment:
Anybody should be able to write new order sizing delegates and have it behave correctly. You should not make special cases here and make two specific sizing delegates have special functions.

Reply:
Hey Martin, the idea is to basically roll these out for single order mode for now. I can make the strategy always pass in the mode and check with it, instead of checking with the name of the delegates.